packages/charting/configure/src/correct-response.jsx

```diff
@@ -97,6 +98,57 @@ const removeCategory = (correctAnswer, data, positionToRemove) => {
   return addCategoryProps(correctAnswerData, data);
 };
 
+export const getUpdatedCategories = (nextProps, prevProps, prevState) => {
+  const nextData = nextProps.model.data || [];
```

Comment:
Everything looks good, but I will insist with sanity check.
Everything that has \. something, has to be checked.
You either make sure nextProps is not null/undefined (if makes sense in your code), like : nextProps = {...nextProps}
Or you check it const nextData = (nextProps && nextProps.model && nextProps.model.data) || [];